ci-operator/config/openshift/oauth-proxy/openshift-oauth-proxy-v4.4-imagestream-1.yaml

```diff
@@ -0,0 +1,35 @@
+base_images:
+  base:
+    cluster: https://api.ci.openshift.org
+    name: "4.4"
+    namespace: ocp
+    tag: base
+binary_build_commands: make build --warn-undefined-variables
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+images:
+- dockerfile_path: Dockerfile.rhel7
+  from: base
+  inputs:
+    bin:
+      as:
+      - registry.svc.ci.openshift.org/ocp/builder:golang-1.13
+      paths: null
+  to: 4.4-imagestream-1
```

Comment:
What does this line mean?

Reply:
tag of the image

Reply:
I don't think we want to update the last `4.4-imagestream-<n>` on every release, do we want to?

Reply:
@sttts i thought we want to... we don't have to, people can just stick to previous release